ERCS/erc-_____.md

```diff
@@ -0,0 +1,342 @@
+---
+eip: ____
+title: "Contactless Crypto Payment Standard"
+description: Standard for initializing contactless payment transactions from EVM wallets
+author: Amhed Herrera, Justin Lee, Arjun Dureja
+discussions-to: 
```

Comment:
Please create a discussions topic in Eth Magicians with a link to this PR.

Reply:
Will do! I'm waiting for the DiscoBot to get me posting privileges. 

This is my user (tied to github username)
https://ethereum-magicians.org/u/amhed/summary

Reply:
Done, thanks again
https://ethereum-magicians.org/t/erc-7798-contactless-payment/21501